pkgs/applications/video/aegisub/default.nix

```diff
@@ -46,71 +46,76 @@ assert alsaSupport -> (alsaLib != null);
 assert pulseaudioSupport -> (libpulseaudio != null);
 assert portaudioSupport -> (portaudio != null);
 
-with lib;
-stdenv.mkDerivation
- rec {
+let
+  inherit (lib) optional;
+in
```

Comment:
This is not really necessary. We can just use lib.optional.

Reply:
It is used four times in a row. It is clearer to use it this way instead of throwing all lib in the scope.

Reply:
just make sure to remove it when it is no longer used.